script.js

```diff
@@ -0,0 +1,229 @@
+
+const TEXTAREA = document.createElement('textarea');
+TEXTAREA.id = 'textarea';
+TEXTAREA.autofocus = true;
+const KEYBOARD = document.createElement('div');
+KEYBOARD.id = 'keyboard';
+const HINT = document.createElement('div');
+HINT.innerText = 'Shift + Alt - change language\n Created under Windows';
+document.body.append(TEXTAREA);
+document.body.append(KEYBOARD);
+document.body.append(HINT);
+let language = sessionStorage.getItem('language') != null ? sessionStorage.getItem('language') : 'en';
+let register = 'lower';
+
+class Button {
+  constructor(htmlParent, keyCode, enLower, enUpper, ruLower, ruUpper) {
+    this.enLower = enLower;
+    this.enUpper = enUpper;
+    this.ruLower = ruLower;
+    this.ruUpper = ruUpper;
+
+    this.htmlKey = document.createElement('div');
+    this.htmlKey.className = 'k-key';
+    this.htmlKey.setAttribute('data', keyCode);
+    this.htmlKey.classList.add(keyCode);
+    this.htmlKey.innerHTML = this.getLangChar(language, register);
+    htmlParent.append(this.htmlKey);
+
+    if (keyCode === 'Backspace' || keyCode === 'Delete' || keyCode === 'Enter' || keyCode === 'ShiftRight') {
+      const nextLine = document.createElement('div');
+      nextLine.className = 'clearfix';
+      htmlParent.append(nextLine);
+    }
+  }
+
+  getLangChar(lang, reg) {
+    if (lang === 'en') {
+      return reg === 'lower' ? this.enLower : this.enUpper;
+    }
+    return reg === 'lower' ? this.ruLower : this.ruUpper;
+  }
+
+  getHtmlElement() {
+    return this.htmlKey;
+  }
+}
+
+const keyMap = {
+  Backquote: new Button(KEYBOARD, 'Backquote', '`', '`', 'ё', 'Ё'),
```

Comment:
Personally I would create constant object for key codes
```
const KEY_CODES = {
   Digit1: 'Digit1',
   Backquote : 'Backquote',
  ...
}
```
and use it 
```
const keyMap = {
  Backquote: new Button(KEYBOARD, KEY_CODES.Backquote, '`', '`', 'ё', 'Ё'),
```

```
 if (keyCode === KEY_CODES.Backspace || ...
```

Reply:
fix in commit 0263963